platform/lib/format-transform/formats/amp4email.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const config = require('@lib/config.js');
+const host = config.getHost(config.hosts.platform);
+
+function absoluteUrl(url) {
+  if (!url.match(/^https?:\/\//)) {
```

Comment:
Use `new URL(url, host).toString()` instead

Reply:
Well TIL. I moved this method to config, let me know if that doesn't make sense.